### PR TITLE
Feature/secciones favs

### PR DESCRIPTION
**Pull request by Diegobr21**
Simple changes:
* Horizontal divs for cards (sections at Index)
* Favorite page now shows cards
* Link to repository

### CODE REVIEW

File: Elecritic/Pages/Favoritos.razor.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Elecritic.Pages {
+    public class Favoritos {
```
Comment (netotz) [inline Elecritic/Pages/Favoritos.razor.cs]: could you please rename the class and both files to `Favorites`?

Comment (netotz) [inline Elecritic/Pages/Favoritos.razor.cs]: could you please rename the class and both files to `MyFavorites` (name suggestion)?

File: Elecritic/Pages/Favoritos.razor.cs [hunk 2]
```diff
@@ -1,9 +1,26 @@
...
+using Elecritic.Services;
+
+using Microsoft.AspNetCore.Components;
+
 namespace Elecritic.Pages {
-    public class Favoritos {
+    public partial class Favoritos {
+
+        [Parameter]
+        public string userId { get; set; }
```
Comment (netotz) [inline Elecritic/Pages/Favoritos.razor.cs]: I suggest not to use a parameter for this page, but to implement a method or so in the logic to determine if a user is logged in by using the singleton of `UserService`.

**Review by netotz — CHANGES_REQUESTED** [review]

File: Elecritic/Pages/Favoritos.razor.cs (hunk 2 above)
Comment (Diegobr21) [reply Elecritic/Pages/Favoritos.razor.cs]: > I suggest not to use a parameter for this page, but to implement a method or so in the logic to determine if a user is logged in by using the singleton of `UserService`.

Actually I did not intend to leave it like that, so yeah, consider that done

**Review by netotz — APPROVED** [review]
great work bro!

**Pull request merged**